auto-entities.js

```diff
@@ -12,7 +12,11 @@ class AutoEntities extends cardTools.litElement() {
     this.card = cardTools.createCard(Object.assign({entities: this.entities}, config.card));
   }
 
-
+ getCardSize() {
+   if(this.card)
+      return typeof this.card.getCardSize === "function" ? this.card.getCardSize() : 1;
```

Comment:
not sure why this isn't just `if (this.card && typeof this.card.getCardSize === 'function')`

Reply:
Sure!

Updated the pull request with the suggestion